exams/utils.py

```diff
@@ -88,6 +89,29 @@ def _has_paid_for_exam(mmtrack, edx_course_key):
     return suppress_payment_for_exam or mmtrack.has_paid(edx_course_key)
 
 
+def _has_passed_course(mmtrack, edx_course_key):
```

Comment:
this is class method, here we suppress payment check if `SUPPRESS_PAYMENT_FOR_EXAM=True`